src/main/java/org/eln2/mc/common/CommonEvents.kt

```diff
@@ -38,4 +50,53 @@ object CommonEvents {
             }
         }
     }
+
+    @SubscribeEvent
+    fun onEntityJoinedWorld(event: EntityJoinWorldEvent) {
+        //Warn new users that we collect analytics
+        if (event.world.isClientSide && event.entity is Player) {
+            if (Eln2.config.enableAnalytics)  {
+                val acknowledgementFile = FMLPaths.CONFIGDIR.get().resolve("ElectricalAge/acknowledgements.yml").toFile()
```

Comment:
Hmm. We're already putting things in `./config/electrical-age.yml` I think; should we have it as a similarly named file? Also, neat function.

Reply:
Personally I think that any config files other than the main config (and sometimes even that) should go into a folder named for the mod. Perhaps a change from `ElectricalAge` to `electrical-age` would be warranted though, assuming that's the actual mod ID?

Reply:
lol, the modid is actually `eln2`. I don't know that modid's can actually have a dash in them due to Forge/Minecraft's ridiculous naming requirements. I've been calling it `electrical-age` and other names since `eln` isn't really a meaningful shortening for electrical age.

At some point we should probably review all of the names and make some decisions.